.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,16 @@
+# This triggers the Build workflow in whatwg/whatwg.org for changes here.
+name: Dispatch
+on:
+  push:
+    branches:
+    - master
+jobs:
+  dispatch:
+    name: Dispatch
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: peter-evans/repository-dispatch@v1
```

Comment:
I guess Dependabot monitors these too?

Reply:
One can configure Dependabot to bump GitHub Actions, yes. I haven't tried it, though.